pkg/database/import.go

```diff
@@ -0,0 +1,88 @@
+package database
+
+import (
+	"fmt"
+
+	"github.com/rotisserie/eris"
+	"gorm.io/gorm"
+	"gorm.io/gorm/clause"
+)
+
+// Import will copy the contents of input db to output db.
+func Import(input, output *DbInstance, dryRun bool) error {
+
+	in := input.DB
+	if err := output.DB.Transaction(func(out *gorm.DB) error {
+
+		if err := importTable(in, out, dryRun, Experiment{}); err != nil {
+			return err
+		}
+		if err := importTable(in, out, dryRun, ExperimentTag{}); err != nil {
+			return err
+		}
+		if err := importTable(in, out, dryRun, Run{}); err != nil {
+			return err
+		}
+		if err := importTable(in, out, dryRun, Tag{}); err != nil {
+			return err
+		}
+		if err := importTable(in, out, dryRun, Metric{}); err != nil {
+			return err
+		}
+		if err := importTable(in, out, dryRun, LatestMetric{}); err != nil {
+			return err
+		}
+		if err := importTable(in, out, dryRun, Param{}); err != nil {
+			return err
+		}
+		if err := importTable(in, out, dryRun, Dashboard{}); err != nil {
+			return err
+		}
+		if err := importTable(in, out, dryRun, App{}); err != nil {
+			return err
+		}
```

Comment:
maybe make this a loop over a slice of models?

Reply:
I tried that but couldn't get the types to work right! I can take another stab at it

Reply:
I don't think Go has any ability to deal with polymorphism that a slice of models would require

Reply:
oh well!